README.md

```diff
@@ -30,24 +29,31 @@ When you export this image in python using pillow library (PIL.Image.open(file_n
 * 'alt_fin_ray': [254, 102, 204],
 * 'trunk': [0, 124, 124]
 
-The approach that we use for extracting traits is the following:
 
+## 2- Presence Absence of Traits
+
+The approach that we use for checking presence and absence of traits is the following:
 
   1. Isolate indivual traits (e.g., isolate the dorsal_fin)
-  2. Remove small blobs and fill in gaps within each trait
-  3. Identify landmarks (defined in section 2)
-  4. Use landmarks and morphological tools (centroid, area, etc.) to extract the measurements (**external characters**)
+  2. Count number of blobs
+  3. Calculate percentage of large blob
```

Comment:
Calculate percentage area of the biggest blod over the total blobs of a particular trait

Reply:
will make branch to fix readme